src/ol/interaction/drawinteraction.js

```diff
@@ -715,6 +715,32 @@ ol.interaction.Draw.prototype.abortDrawing_ = function() {
 
 
 /**
+ * Continue drawing a feature.  This only works on line features, that is
+ * features with a `LINE_STRING` geometry.
```

Comment:
To users, the draw mode (and geometry types) look like `'LineString'` (instead of `LINE_STRING`).  If you change this, I think this doc would be more informative with something like:

> Extend an existing geometry by drawing additional points.  This only works on features with `'LineString'` geometries, where the interaction will extend lines by adding points to the end of the coordinates array.